proto/ibc/applications/transfer/v1/query.proto

```diff
@@ -35,6 +35,11 @@ service Query {
   rpc EscrowAddress(QueryEscrowAddressRequest) returns (QueryEscrowAddressResponse) {
     option (google.api.http).get = "/ibc/apps/transfer/v1/channels/{channel_id}/ports/{port_id}/escrow_address";
   }
+
+  // TotalNativeIBCOut returns the total native token that's IBC'd out.
+  rpc TotalNativeIBCOut(QueryTotalNativeIBCOutRequest) returns (QueryTotalNativeIBCOutResponse) {
+    option (google.api.http).get = "/ibc/apps/transfer/v1/total_native_ibc_out/{denom}";
```

Comment:
Maybe for naming consistency and in case in the future we want to have other queries related to denoms?

```suggestion
    option (google.api.http).get = "/ibc/apps/transfer/v1/denoms/{denom}/total_escrow";
```

And would this work fine if the denom contains slashes?

Reply:
Wondering also if the `port_id` should be part of the URI...

Reply:
Since the tracking is only happening for the transfer port, I think it makes sense to keep the port out of the uri or hard-code it to "transfer".